startables/startables.py

```diff
@@ -454,6 +454,8 @@ class Bundle:
      * are understood as having a common context, in particular when evaluating expressions.
     """
 
+    MIN_BUNDLE_SEPARATOR = 3
```

Comment:
Hey Heidi, Let's have a chat?
Why 3?
Also, I have a funny feeling about this more generally... seems to me, we have a bug in the reader, but we're fixing the writer... It's a fine workaround for cases where we're reading files that this same package wrote in the first place; but won't help when reading files created otherwise

Reply:
Ok, you didn't insert this 3... someone else did, dunno who. You just refactored it. Ok.